lib/lambda_ethereum_consensus/p2p/incoming_requests/handler.ex

```diff
@@ -128,6 +128,35 @@ defmodule LambdaEthereumConsensus.P2P.IncomingRequests.Handler do
     end
   end
 
+  defp handle_req("beacon_blocks_by_root/2/ssz_snappy", message_id, message) do
+    with <<24, snappy_blocks_by_root_request::binary>> <- message,
```

Comment:
We should handle this case with an error. For example:

```elixir
    with {:ok, snappy_blocks_by_root_request} <- parse_message_size(message),
```

```elixir
  defp parse_message_size(<<24, request::binary>>), do: {:ok, request}
  defp parse_message_size(_), do: {:error, "invalid request"}
```

That way the result is always `:ok` or `{:error, reason}`